### ISSUE TITLE
Remove Sass from Storybook

### ISSUE THREAD

**Issue by xdrdak** [lightspeed/flame#1]
## Description

Remove all Sass references and usage of it from our stories.

Since we want to promote the usage of `emotion`, it might make more sense to build custom components using `emotion` + our theme values than pulling stuff from our legacy generated Sass values.

This will also enable us to remove some rather pointless plugins to bundle css and sass into the final storybook build.

**Comment by glambert** [comment]
Partially closed by #7, correct?

**Comment by alexismo** [comment]
`flame/stories/styles/stories.scss` still uses a bunch of Sass

**Comment by glambert** [comment]
> `flame/stories/styles/stories.scss` still uses a bunch of Sass

Yup, "partially" is the keyword 😄This still needs to be done, wanna pick up @alexismo? 😉
